system-addon/content-src/components/Card/Card.jsx

```diff
@@ -56,14 +56,18 @@ class Card extends React.Component {
     const isContextMenuOpen = this.state.showContextMenu && this.state.activeCard === index;
     // Display "now" as "trending" until we have new strings #3402
     const {icon, intlID} = cardContextTypes[link.type === "now" ? "trending" : link.type] || {};
+    const hasImage = link.image || link.imageLoading;
```

Comment:
With `imageLoading` renamed to `hasImage`, I think we should just rely on that value instead of additionally checking `image`, so `const {hasImage} = link;`

Reply:
Oh. Although I see you probably did this for backwards compatibility for other things passing in just `image`…

Reply:
Yeah because pocket for example doesn't set hasImage